components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -101,6 +102,9 @@ void AdsServiceFactory::RegisterProfilePrefs(
   #endif
 
   registry->RegisterIntegerPref(prefs::kBraveAdsIdleThreshold, 15);
+  registry->RegisterBooleanPref(prefs::kBraveAdsShowAdsNotification, true);
+  registry->RegisterTimePref(
+      prefs::kBraveAdsLaunchNotificationTimestamp, base::Time());
```

Comment:
Is it intentional to set time to NULL. If you require the current time you can use Time::Now()